rule-packs/azure.json

```
@@ -0,0 +1,387 @@
+[
+    {
+      "name": "azure-asc-monitor-system-updates",
+      "description": "Finds if Azure Security Center setting 'Monitor System Updates' is disabled.",
```

I think of queries as returning collections of assets, and alerts are monitoring assets, so what is found are assets. This is worded as though it will return a `boolean` value. How about:
```suggestion
      "description": "Finds Azure Security Center policies where 'Monitor System Updates' is disabled.",
```